api/src/main/resources/messages.properties

```diff
@@ -17,6 +17,7 @@ sync2.configuration.import.file.label=Choose file with configuration
 sync2.configuration.errors.invalidFile=Invalid file
 sync2.configuration.errors.invalidJson=Invalid Json
 sync2.configuration.errors.serverError=Server Error
+sync2.configuration.parentUrl.notValid=The parentFeedLocation URI wasn't set correctly
```

Comment:
It depends what we need. In scope of this ticket we should verify if URI is not empty, so putting value here has no sense. However, if we want to verify if URI is valid, putting value is a good idea

Reply:
I agree with @kmadej it isn't in the scope of this ticket. But I rename this message property to "sync2.configuration.parentUrl.empty". I believe it will be more obvious.